composer.json

```diff
@@ -15,12 +15,13 @@
         }
     },
     "require": {
-        "illuminate/support": "5.8.*|^6.0",
+        "php": "7.4 - 8.3",
+        "illuminate/support": "5.8.*|^6.0|^10.0",
         "guzzlehttp/guzzle": "^6.3|^7.0",
-        "illuminate/validation": "5.8.*|^6.0",
-        "propaganistas/laravel-phone": "^4.2",
-        "illuminate/console": "5.8.*|6.*",
-        "psr/log": "^1.1"
+        "illuminate/validation": "5.8.*|^6.0|^10.0",
```

Comment:
Maybe change this to double pipe for `OR` ?

```
A double pipe (||) will be treated as a logical OR

Note: In older versions of Composer the single pipe (|) was the recommended alternative to the logical OR. Thus for backwards compatibility the single pipe (|) will still be treated as a logical OR.
```

Reply:
Done, thanks 👍